spec/customer_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'customer'
+
+describe Customer do
+
+  let(:menu) { DishesList.new }
+  let(:customer) { Customer.new(menu) }
+
```

Comment:
The described class should be the only real object and everything else should be mocked.